client/hotrod-client/src/test/java/org/infinispan/client/hotrod/AuthenticationTest.java

```diff
@@ -40,7 +40,7 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
       return cacheManager;
    }
 
-   private ConfigurationBuilder initServerAndClient() {
+   protected ConfigurationBuilder initServerAndClient() {
```

Comment:
Sorry - had a new class test extending this one, which was not included in the commit.  Added.